server/src/main/java/org/jboss/as/server/deployment/DeploymentHandlerUtil.java

```diff
@@ -334,4 +351,21 @@ private static String getFormattedFailureDescription(OperationContext context) {
         }
         return failureDesc;
     }
+
+    public static boolean shouldNotify(OperationContext context, ModelNode operation) throws IllegalArgumentException {
+        if(context.isBooting()) {
+            return false;
+        }
+        if (operation.hasDefined(OWNER.getName())) {
```

Comment:
I wanted to avoid the scanner emitting a notification to be processed by the scanner